app/page.tsx

```diff
@@ -14,13 +14,13 @@ export default async function Page() {
   });
   return (
     <div className="mx-auto bg-slate-100 pt-8">
-      <div className="max-w-4xl mx-auto py-8">
+      <div className="global-width mx-auto py-8">
         <WelcomeTexts
           title="Welcome to Mumble"
           description="Did you hear that? They've shut down the main reactor."
         />
       </div>
-      <div className="max-w-4xl mx-auto">
+      <div className={`global-width mx-auto`}>
```

Comment:
is this template literal necessary here?

Reply:
no, it is not. I was trying to include it dynamically from the configFrontend. it does not work ;-)